### DISCUSSION TITLE
Could I fork this font to make a Latin/Greek/Cyrillic-only version?

### DISCUSSION THREAD

**Discussion by cssobral2013** [orioncactus/pretendard#17] [Q&A] [answered]
Note: Keeping your copyright notes, and adding my ones, of course.

**Comment by orioncactus** [comment]
Sure. Pretendard is based on [Inter](https://github.com/rsms/inter) and [Source Han Sans](https://github.com/adobe-fonts/source-han-sans) and it's distributed under the [Open Font License 1.1](https://github.com/orioncactus/pretendard/blob/main/LICENSE), so you can do whatever you want.

But if you want to use the non-CJK subset version of Pretendard on web, although it only supports Basic Latin that you’ve mentioned, you can use the ‘Dynamic Subset’ of Pretendard:

#### CSS

```css
@import url('https://cdn.jsdelivr.net/gh/orioncactus/pretendard/dist/web/static/pretendard-dynamic-subset.css');
```

#### HTML

```html
<link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard/dist/web/static/pretendard-dynamic-subset.css" />
```

But it's the version with only the Basic set of the whole Latin character set among those you've mentioned, so I'm considering including Latin/Greek/Cyrillic into the current dynamic subset. I hope this comment is helpful for you. Thank you.

**Comment by cssobral2013** [comment]
@orioncactus - Good, however I'm wishing to use this on desktop.

**Comment by orioncactus** [comment]
Then you can use the latest Pretendard release. It includes all of the Latin/Greek/Cyrillic character set. Can you tell me why you want the non-CJK subset version of Pretendard?

**Comment by cssobral2013** [comment]
@orioncactus - Because it looks better than original Inter (it looks closer
to San Francisco than original Inter).

Em qua., 14 de jul. de 2021 1:54 AM, Kil Hyung-jin ***@***.***>
escreveu:

**Comment by orioncactus** [comment]
Thank you for your compliment. I plan on consistently improving Pretendard to become a more general font.

I'll close this issue. And if you have another question, feel free to reopen this issue!

**Comment by frencojobs** [comment]
Hi @orioncactus! Is there a Latin/Greek/Cyrillic only version for "Pretendard Variable"? Because I'm trying to use Pretendard on my website and the `woff2` variable font file provided from here is about **2.3mb**, which I think is too big.

**Reply by orioncactus** [reply]
Hello @frencojobs! Thank you for your interest about Pretendard. In the near future, Pretendard will be updated by adding a new family called 'Pretendard JP' that's supporting Japanese. So that means It can expand additional family like 'Pretendard Std' as Latin/Greek/Cyrillic only version, but since it is not currently a priority, I'm not sure when exactly I can release Latin/Greek/Cyrillic only version. So in the meantime, I recommend you to make subset file from Pretendard Variable using [fonttools](https://fonttools.readthedocs.io/en/latest/subset/index.html).

**Comment by orioncactus** [comment, marked as answer]
Hello! I just released Latin/Greek/Cyrillic only version of Pretendard and Pretendard Variable 2 days ago with named 'Pretendard Std', and also you can use the Pretendard webfonts with low-weight as 'Pretendard Std' via jsDelivr. 

Here's the code:

---

## Full version of Pretendard Std

- font-family name: `'Pretendard Std'`

#### HTML

```html
<link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard/dist/web/static/pretendard-std.css" />
```

#### CSS

```css
@import url('https://cdn.jsdelivr.net/gh/orioncactus/pretendard/dist/web/static/pretendard-std.css');
```

## Dynamic subset version of Pretendard Std like Roboto from Google Fonts

- font-family name: `'Pretendard Std'`

#### HTML

```html
<link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard/dist/web/static/pretendard-std-dynamic-subset.css" />
```

#### CSS

```css
@import url('https://cdn.jsdelivr.net/gh/orioncactus/pretendard/dist/web/static/pretendard-std-dynamic-subset.css');
```

## Variable version of Pretendard Std

- font-family name: `'Pretendard Std Variable'`

#### HTML

```html
<link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard/dist/web/variable/pretendardvariable-std.css" />
```

#### CSS

```css
@import url('https://cdn.jsdelivr.net/gh/orioncactus/pretendard/dist/web/variable/pretendardvariable-std.css');
```

FYI: @cssobral2013 @frencojobs
